src/main/kotlin/lotto/LottoController.kt

```diff
@@ -1,24 +1,54 @@
 package lotto
 
-import lotto.domain.LottoResult
-import lotto.domain.LottoStore
+import lotto.domain.AutoStrategy
+import lotto.domain.LottoGame
+import lotto.domain.LottoNumber
+import lotto.domain.LottoTicket
+import lotto.domain.Lottoes
+import lotto.domain.ManualStrategy
+import lotto.domain.Money
 import lotto.domain.WinningLotto
 import lotto.ui.InputView
 import lotto.ui.OutputView
 
+private val inputView = InputView()
+private val outputView = OutputView()
+
 fun main() {
-    val inputView = InputView()
-    val outputView = OutputView()
-    val lottoStore = LottoStore()
-    val lottoResult = LottoResult()
+    val money = Money(inputView.inputMoney())
+    val numberOfManual = inputView.inputNumberOfManual()
+    val lottoGame = LottoGame(money)
+
+    val manualLottoes = inputManualLottoes(numberOfManual, lottoGame) ?: Lottoes(emptyList())
+    val autoLottoes = lottoGame.purchaseLottoes(AutoStrategy())
 
-    val money = inputView.inputMoney()
-    val lottoes = lottoStore.purchaseAuto(money)
+    outputView.printPurchasedLottoes(manualLottoes, autoLottoes)
+    val universalLottoes = Lottoes(manualLottoes.toList() + autoLottoes.toList())
+    val winningLotto = WinningLotto(createWinningTicket(), createBonusNumber())
+
+    outputView.printLottoesResult(
+        money,
+        autoLottoes.getMyLottoesRanks(winningLotto),
+        manualLottoes.getMyLottoesRanks(winningLotto)
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        universalLottoes.getMyLottoesRanks(winningLotto)
```